core/load-git.c

```diff
@@ -989,7 +992,7 @@ struct keyword_action dive_action[] = {
 #define D(x) { #x, parse_dive_ ## x }
 	D(airpressure), D(airtemp), D(buddy), D(cylinder), D(divemaster), D(divesiteid), D(duration),
 	D(gps), D(location), D(notes), D(notrip), D(rating), D(suit),
-	D(tags), D(visibility), D(watertemp), D(weightsystem)
+	D(tags), D(watersalinity), D(visibility), D(watertemp), D(weightsystem)
```

Comment:
please look at the comment in line 989 (new line numbers) above., These need to be sorted or the code won't work.

Reply:
I will correct this. There is actually a comment in the code that the items need to be in sorted order. I will wait for the comments from others so that I can consolidate them into a single commit. I also need to add translation of strings and remove unwanted #includes. As is at the moment, visibility does not work.

Reply:
Done